dynawo/sources/Modeler/DataInterface/DYNComponentInterface.cpp

```diff
@@ -111,6 +124,30 @@ ComponentInterface::getType() const {
   return type_;
 }
 
+const std::string&
+ComponentInterface::getTypeAsString() const {
+  static const std::string stringTypes[] = {
+      "UNKNOWN",
+      "BUS",
+      "CALCULATED_BUS",
+      "SWITCH",
+      "LOAD",
+      "LINE",
+      "GENERATOR",
+      "SHUNT",
+      "DANGLING_LINE",
+      "TWO_WTFO",
```

Comment:
please use TWO_WINDINGS_TRANSFORMER and THREE_WINDINGS_TRANSFORMER

Reply:
Done